server/constants.py

```diff
@@ -5,6 +5,13 @@
 STAFF_ROLE = 'staff'
 INSTRUCTOR_ROLE = 'instructor'
 LAB_ASSISTANT_ROLE = 'lab assistant'
+ROLES_NEW={
+'student':'Student',
+'grader':'Reader',
+'staff':'Teaching Assistant',
```

Comment:
We should reuse `STAFF_ROLE` from above instead of repeating the string literal `'staff'`